docs/reference/logging/index.md

```diff
@@ -25,6 +25,16 @@ container's logging driver. The following options are supported:
 
 The `docker logs`command is available only for the `json-file` logging driver.  
 
+### The json-file options
+
+The following logging options are supported for the `json-file` logging driver:
+
+   --log-opt max-size=[0-9+][k/m/g/K/M/G]
+   --log-opt max-file=[0-9+]
+
+`max-size` sets the size of each of the log files that are rolled over
+`max-file` sets the maximum number of log files that are rolled over amongst
```

Comment:
missing something after "amongst" --- also --- what is the point of the max size?  and 9+ does that mean unlimited?  Why aren't we use date or something --- roll over files that are aged past 9 days and x size...

---> Something like

### The json-file options

The `json-file` logging driver supports the following options:

```
--log-opt max-size=[0-9+][k/m/g/K/M/G]
--log-opt max-file=[0-9+]
```

`max-size` Logs that reach this size are rolled over.  You can set the size in kilobytes (k), megbytes (m), or gigbytes (g).